tests/cli/test_felix.py

```diff
@@ -36,9 +47,6 @@ def test_run_felix(click_runner,
     # Main thread (Flask)
     os.environ['NUCYPHER_FELIX_DB_SECRET'] = INSECURE_DEVELOPMENT_PASSWORD
 
-    # Mock live contract registry reads
-    LocalContractRegistry.read = lambda *a, **kw: test_registry.read()
-
```

Comment:
This mock was not properly restored to the original call - causing some pretty odd CI failures for the tests run in the same container as this test, after this one finishes.